gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/commands/AbstractHystrixCommand.groovy

```diff
@@ -49,7 +53,11 @@ abstract class AbstractHystrixCommand<T> extends HystrixCommand<T> {
 
   @Override
   protected T run() throws Exception {
-    return work()
+    try {
+      return work()
+    } catch (RetrofitError error) {
+      throw UpstreamBadRequest.classifyError(error, [HttpStatus.TOO_MANY_REQUESTS.value()])
```

Comment:
Was going to do that when I moved it to kork-hystrix but I've gone ahead and done it here now.

    throw UpstreamBadRequest.classifyError(error, HttpStatus.values().findAll { it.is4xxClientError() }*.value() as Collection<Integer>)